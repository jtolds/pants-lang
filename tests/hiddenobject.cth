
print "hidden object test"

{
  print .x
}(|x:3)

print .x
