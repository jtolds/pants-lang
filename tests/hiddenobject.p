
println "hidden object test"

{
  println .x
}(;;x:3)

println .x # should fail
