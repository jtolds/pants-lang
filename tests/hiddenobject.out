hidden object test
3
field u_x_42 not found!

return code: 1
